pkg/users/account-create-dialog.js

```diff
@@ -116,6 +116,23 @@ function validate_username(username, accounts) {
     return null;
 }
 
+function is_valid_char_real_name(c) {
+    return (c >= 'a' && c <= 'z') ||
+        (c >= 'A' && c <= 'Z') ||
+        (c >= '0' && c <= '9') ||
+        c == '.' || c == '_' || c == '-' || c == ' ';
+}
+
+function validate_real_name(real_name) {
+    if (!real_name)
+        return _("No real name specified");
+
+    for (let i = 0; i < real_name.length; i++) {
```

Comment:
Javascript has a nicer way to do it:
real_name.every(is_valid_char_real_name)

Reply:
Oh that is much nicer! Will update my PR and only check for the delimiter. Thanks!